cmake/xnnpack/CMakeLists.txt

```diff
@@ -0,0 +1,4975 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
What is the future maintenance plan for this file? If this is a modified file, is it possible to use a patch so that we can track our changes more clearly.